atomics/include/desul/atomics/Fetch_Op_Generic.hpp

```diff
@@ -89,8 +89,7 @@ DESUL_IMPL_ATOMIC_FETCH_OP_SHIFT_HOST_AND_DEVICE(rshift)
   template <class T, class MemoryOrder, class MemoryScope>                     \
```

Comment:
```suggestion
  // NOTE: using atomic_oper_fetch in the fallback implementation of atomic_store
  // to avoid reading potentially uninitialized values which would yield
  // undefinied behavior.
  template <class T, class MemoryOrder, class MemoryScope>                     \
```